packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -188,6 +196,34 @@ export default class TelecomPackMigrationOffersCtrl {
     );
   }
 
+  initializeGtrOptions() {
+    this.process.migrationOffers.result.offers.forEach((offer) => {
+      let gtrOptions = [];
+      // Initialize GTR options with a none value
+      const init = {
+        name: this.GTR_NONE,
+        label: this.$translate.instant('telecom_pack_migration_no_gtr'),
+      };
+      gtrOptions.push(init);
+
+      Object.values(offer.options).forEach((option) => {
+        if (option.name.startsWith('gtr_')) {
+          const gtrOption = option;
+          gtrOption.label = this.$translate.instant(
+            `telecom_pack_migration_${gtrOption.name}`,
+            { price: gtrOption.optionalPrice.text },
+          );
+          gtrOptions.push(gtrOption);
+        }
+      });
+
+      if (gtrOptions.length === 1) {
+        gtrOptions = [];
+      }
+      set(offer, 'gtrOptions', gtrOptions);
+    });
```

Comment:
**nitpick:** Please use `Array.prototype.map` for altering an array. In this snippet, each array item is overridden inside a `forEach` which makes it less readable.